scripts/travis/setup-mistral.sh

```diff
@@ -6,8 +6,39 @@ if [ "$(whoami)" != 'root' ]; then
     exit 2
 fi
 
-MISTRAL_STABLE_BRANCH="st2-0.9.0"
+function version_ge() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)" == "$1"; }
+
+# Determine which mistral version to use
+if [[ "${TRAVIS_BRANCH}" == 'master' ]]; then
```

Comment:
What can I do here?

Refactor `st2_deploy.sh` to re-use ssh functions as mentioned in https://github.com/StackStorm/st2/pull/1778#issue-98418711 ?

If so, isn't `st2_deploy.sh` to be deprecated soon?